src/__mocks__/uuid.js

```diff
@@ -1,3 +0,0 @@
-module.export = {
-  v1: jest.fn(() => 'mocked-uuid-v1'),
-};

```

Comment:
This didn't actually work, since we needed to mock `uuid/v1`, not an export named `v1` on `uuid`. Since removing it didn't cause any tests to fail, I just decided to leave it deleted.